src/wallet/kasWallet.js

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright 2020 The caver-js-ext-kas Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the “License”);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an “AS IS” BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * The wallet class that uses the KAS Wallet API.
+ * @class
+ */
+class KASWallet {
+    /**
+     * Creates a wallet instance that uses the KAS Wallet API. <br>
+     * @example <caption>If you want to use KASWallet as a substitute for an in-memory wallet, you should pass last parameter in constructor. Please refer to the example below.</caption>
+     * const caver = new Caver(chainId, accessKeyId, secretAccessKey, true) // Use KASWallet
+     *
+     * const caver = new Caver(chainId, accessKeyId, secretAccessKey, false) // Use in-memory wallet
```

Comment:
Didn't we decide to remove the boolean variable for the activation of KASWallet?

Reply:
This comments are old; I removed that part.